reference/docs-conceptual/learn/remoting/WinRMSecurity.md

```diff
@@ -1,22 +1,24 @@
 ---
-ms.date:  06/05/2017
+ms.date: 06/11/2020
 keywords:  powershell,cmdlet
 title:  WinRMSecurity
 ---
 
 # PowerShell Remoting Security Considerations
 
-PowerShell Remoting is the recommended way to manage Windows systems. PowerShell Remoting is enabled by default in Windows Server 2012 R2. This document covers security concerns,
-recommendations, and best practices when using PowerShell Remoting.
+PowerShell Remoting is the recommended way to manage Windows systems. PowerShell Remoting is enabled
+by default in Windows Server 2012 R2. This document covers security concerns, recommendations, and
+best practices when using PowerShell Remoting.
 
 ## What is PowerShell Remoting?
 
-PowerShell Remoting uses [Windows Remote Management (WinRM)](https://msdn.microsoft.com/library/windows/desktop/aa384426.aspx), which is the Microsoft implementation of the
-[Web Services for Management (WS-Management)](https://www.dmtf.org/sites/default/files/standards/documents/DSP0226_1.2.0.pdf) protocol, to allow users to run PowerShell commands on remote
-computers. You can find more information about using PowerShell Remoting at [Running Remote Commands](https://technet.microsoft.com/library/dd819505.aspx).
+PowerShell Remoting uses [Windows Remote Management (WinRM)](https://msdn.microsoft.com/library/windows/desktop/aa384426.aspx),
```

Comment:
Let's fix these old URLs.  This one goes to https://docs.microsoft.com/windows/win32/winrm/portal

Use site relative links and fix any others in the doc.